easybuild/easyconfigs/n/ncurses/ncurses-5.9-ictce-5.4.0.eb

```diff
@@ -0,0 +1,16 @@
+easyblock = 'ConfigureMake
```

Comment:
no (for now), there's (still) an easyblock for ncurses (until we've figured out what to do with https://github.com/hpcugent/easybuild-easyblocks/pull/542